terraform/grafana/Makefile

```
@@ -0,0 +1,43 @@
+
+
+ENV := $(shell cat .last_used_env || echo "not-set")
+ENV_FILE := $(PWD)/.env.${ENV}
```

Use env from top level, otherwise it would be easy to get it wrong
```suggestion
ENV := $(shell cat ../.last_used_env || echo "not-set")
-include ../.env.${ENV}
```